api/src/changeRequests/processChangeRequest.ts

```diff
@@ -46,6 +46,10 @@ export async function processChangeRequest(
             doc.memberOf = parentDoc.memberOf;
             doc.tags = parentDoc.tags;
             doc.image = parentDoc.image;
+
+            if (parentDoc.type == DocType.Tag) {
+                doc.tagType = (parentDoc as TagDto).tagType;
```

Comment:
I kept the name the same as the parent document as these are replicated fields from the parent (together with other fields - see [add github permalink to "processChangeRequest.ts line 40]

If we should change the naming for better readability / clarity, we should do it for all the replicated fields.

Reply:
Ah, right. I noticed `parentType` and `parentId`, which are both prefixed. It makes a little bit more sense for replicated fields.

> If we should change the naming for better readability / clarity, we should do it for all the replicated fields.

Maybe that would make sense? I feel it could be confusing, especially for cases where we have similar names, like now we replicate `image` to the content, which could clash in naming with a localised image.

Reply:
Updated replicated fields to be prefixed with "parent", with the exception of the memberOf field as this is needed to query data from the database for given group ID's for most document types.